storages/backends/gcloud.py

```diff
@@ -83,7 +83,7 @@ def close(self):
             if self._is_dirty:
                 self.blob.upload_from_file(
                     self.file, rewind=True, content_type=self.mime_type,
-                    predefined_acl=self._storage.default_acl)
+                    predefined_acl=self._storage.object_parameters.get('acl', self._storage.default_acl))
```

Comment:
Should this call ‘get_object_parameters’?

Reply:
I haven't grasp how all things tie together and didn't want to make a bigger change here, especially without a testcase.
I've pushed an update.